silkworm/silkrpc/core/blocks.cpp

```diff
@@ -61,8 +61,8 @@ Task<uint64_t> get_block_number_by_tag(const std::string& block_id, const rawdb:
     co_return block_number;
 }
 
-Task<std::pair<uint64_t, bool>> get_block_number(const std::string& block_id, const rawdb::DatabaseReader& reader, bool latest_required) {
-    uint64_t block_number;
+Task<std::pair<BlockNum, bool>> get_block_number(const std::string& block_id, const rawdb::DatabaseReader& reader, bool latest_required) {
```

Comment:
The same change should be applied also in `blocks.hpp`

Reply:
ok